Makefile

```
@@ -1,9 +1,27 @@
 .PHONY: all
-all: 
-
+all:
+
+
+.PHONY: certs
+certs:
```

checkmake (0.2.2)

[warning] Target body for "certs" exceeds allowed length of 5 (15).

(maxbodylength)